src/HexagonAlignLoads.h

```diff
@@ -0,0 +1,22 @@
+#ifndef HALIDE_HEXAGON_ALIGN_LOADS_H
+#define HALIDE_HEXAGON_ALIGN_LOADS_H
```

Comment:
Is there anything Hexagon specific about this? It's fine to keep it in this branch, but unless it's really hexagon specific, we should just make this something general.